### DISCUSSION TITLE
可以在右上角油猴插件展示界面自定义决定左侧是否显示红能和设置工具栏

### DISCUSSION THREAD

**Discussion by mcyangky1** [the1812/Bilibili-Evolved#4860] [功能建议 / Ideas]
![987fd549-ecb1-4a39-8f13-1ba3eb398347](https://github.com/user-attachments/assets/cdf711ed-9fd7-4b8e-8a7f-f55c8b7c2429)
就像这张图片这样，在看直播时暂时无法自动隐藏，请问能否加上这个快捷开关

**Comment by Tinhone** [comment]
可以试一试组件`自动隐藏侧栏`，选项里`触发区域宽度`设置为 1

**Reply by mcyangky1** [reply]
谢谢，我去试一下

**Reply by mcyangky1** [reply]
谢谢，已解决
